FreeType.xs

```diff
@@ -628,7 +628,7 @@ qefft2_face_fixed_sizes (Font_FreeType_Face face)
         int i;
         FT_Bitmap_Size *size;
         HV *hash;
-        double pt, ppem;
+        double pt = 12.0, ppem;
```

Comment:
All uses of  `pt` are protected by `if(size->size)`, hence it will be initialized when used. So, no need of defaulting to 12.